airbyte-cdk/java/airbyte-cdk/db-sources/src/testFixtures/java/io/airbyte/cdk/integrations/source/jdbc/test/JdbcSourceAcceptanceTest.java

```diff
@@ -285,7 +285,7 @@ public void setup() throws Exception {
     });
   }
 
-  protected void maybeSetShorterConnectionTimeout() {
+  protected void maybeSetShorterConnectionTimeout(final JsonNode config) {
```

Comment:
Signature was changed to reflect that of the similar method in the new `JdbcSourceTest` fixture.